Block/Product/Bundle.php

```diff
@@ -75,6 +75,20 @@ public function __construct(
         );
     }
 
+    public function toHtml()
+    {
+        $aboveThreshold = $this->dataHelper->aboveThreshold(
+            $this->getProduct()->getPriceInfo()->getPrice('final_price')->getValue()
```

Comment:
I don't have my M2 fully setup again so I can't check this myself yet, but I think I remember 'final_price' coming in as 0 on bundled products in M1 in some/all cases, do you know if this is the case here too or it's just an M1 thing?

Reply:
Need to do it this way, because it's dynamic price product, so $product->getPrice() would return 0, didn't see any problems with it on local env